neo/rawio/intanbinaryrawio.py

```diff
@@ -0,0 +1,442 @@
+        chan_info["sampling_rate"] = sr / BLOCK_SIZE
+        chan_info["units"] = "V"
+        chan_info["gain"] = 0.0000748
+        chan_info["offset"] = 0.0
+        ordered_channels.append(chan_info)
+        data_dtype[2] += [(name, "uint16",)]
+
+    # temperature is not an official channel in the header
+    for i in range(global_info["num_temp_sensor_channels"]):
+        name = "temperature_{}".format(i)
+        chan_info = {"native_channel_name": name, "signal_type": 20}
+        chan_info["sampling_rate"] = sr / BLOCK_SIZE
+        chan_info["units"] = "Celsius"
+        chan_info["gain"] = 0.001
+        chan_info["offset"] = 0.0
+        ordered_channels.append(chan_info)
+        data_dtype += [(name, "int16",)]
+
+    # 3: USB board ADC input channel stored in analogin.dat/board-ANALOG-*
+    for chan_info in channels_by_type[3]:
+        name = chan_info["native_channel_name"]
+        chan_info["sampling_rate"] = sr
+        chan_info["units"] = "V"
+        if global_info["eval_board_mode"] == 0:
+            chan_info["gain"] = 0.000050354
+            chan_info["offset"] = 0.0
+        elif global_info["eval_board_mode"] == 1:
+            chan_info["gain"] = 0.00015259
+            chan_info["offset"] = -32768 * 0.00015259
+        elif global_info["eval_board_mode"] == 13:
+            chan_info["gain"] = 0.0003125
+            chan_info["offset"] = -32768 * 0.0003125
+        ordered_channels.append(chan_info)
+        data_dtype[3] += [(name, "uint16", BLOCK_SIZE)]
+
+    # 4: USB board digital input channel stored in digitalin.dat/board-DIGITAL-IN-*
+    # 5: USB board digital output channel stored in digitalout.dat/board-DIGITAL-OUT-*
+    for sig_type in [4, 5]:
+        # User can obtain digital channels from analog_signal_chunk and then process
+        # them themself
+        if len(channels_by_type[sig_type]) > 0:
+            name = {4: "DIGITAL-IN", 5: "DIGITAL-OUT"}[sig_type]
+            chan_info = channels_by_type[sig_type][0]
+            chan_info["native_channel_name"] = name  # overwite to allow memmap to work
+            chan_info["sampling_rate"] = sr
+            chan_info["units"] = "TTL"  # arbitrary units so I did TTL for the logic
+            chan_info["gain"] = 1.0
+            chan_info["offset"] = 0.0
+            ordered_channels.append(chan_info)
+            data_dtype[sig_type] += [(name, "uint16", BLOCK_SIZE)]
+
+    if bool(global_info["notch_filter_mode"]) and version >= V("3.0"):
+        global_info["notch_filter_applied"] = True
+    else:
+        global_info["notch_filter_applied"] = False
+
+    # because data_dtype has all possible dtypes we need to delete the dtypes which aren't contained in this dataset
+    # so if the data_dtype value is still an empty list we delete to ignore.
+    dtype_cleanup = []
+    for key, value in data_dtype.items():
+        if len(value)==0:
+            dtype_cleanup.append(key)
+    for key in dtype_cleanup:
+        _ = data_dtype.pop(key)
```

Comment:
@h-mayorquin, this is required otherwise downstream functions will expect every stream to be present. But this was just a quick fix. If you have a better fix happy to do it.